src/Adapter/Cart/CartPresenter.php

```diff
@@ -283,7 +286,20 @@ public function present($cart)
             'id_address_delivery' => $cart->id_address_delivery,
             'id_address_invoice' => $cart->id_address_invoice,
             'vouchers' => $this->getTemplateVarVouchers($cart),
-        ];
+            'minimalPurchaseRequired' =>
+                ($this->priceFormatter->convertAmount($productsTotalExcludingTax) < $minimalPurchase) ?
+                sprintf(
+                    $this->translator->trans(
+                        'A minimum shopping cart total of %s (tax excl.) is required to validate your order.
+                        Current cart total is %s (tax excl.).',
+                        array(),
+                        'Cart'
+                    ),
+                    $this->priceFormatter->convertAndFormat($minimalPurchase),
+                    $this->priceFormatter->convertAndFormat($productsTotalExcludingTax)
+                ) :
+                '',
```

Comment:
Just an idea right now…

As the PSR / Symfony do not control the ternary operator structure, maybe we should follow the AirBnb one?